src/Makefile

```diff
@@ -90,15 +90,17 @@ test:
 
 
 celery:
-	DJANGO_ENV=local celery -A _main_.celery.app worker -l info
+	DJANGO_ENV=local celery -A _main_.celery.app worker -l info -B
 .PHONY: celery 
 
 
 beat:
-	DJANGO_ENV=local celery -A _main_.celery.app beat -l info
+	celery -A _main_.celery.app beat -l info
 
 celery-remote:
 	celery -A _main_.celery.app worker -l info
 
-beat:
-	celery -A _main_.celery.app beat -l info
+
+.PHONY: full-celery-local
+full-celery-local: 
```

Comment:
I don't understand this.  Does this command run everything in one window?  
DId the version which I suggested not work for you?

Reply:
yes. it runs the django server, celery and celery beat in one terminal